Frontend/android-native/app/src/main/java/in/juspay/mobility/utils/RideRequestUtils.java

```diff
@@ -264,4 +269,74 @@ public void openApplication(Context context){
         }catch (Exception ignored){}
     }
 
+    private static void updateStorage(String key, String value, Context context){
```

Comment:
remove this function Just use `sharedPref.edit().putString(storage_key, storage_value).apply();`